flapp-webapp/src/App.vue

```diff
@@ -1,16 +1,21 @@
 <template>
   <div class="app-outer fill-body" id="app">
-    <FlappSurveys></FlappSurveys>
+    <NavigationTopbar />
```

Comment:
FYI although the component is registered as NavigationTopbar, Vue converts that internally to navigation-topbar. This name should be preferred when referencing it from a template as names with hyphens will not conflict with native HTML elements.

Reply:
@anissa-agahchen